tensorboard/webapp/core/effects/core_effects.ts

```diff
@@ -172,11 +171,54 @@ export class CoreEffects {
         })
       );
 
-      const runsReload$ = this.onDataReload$.pipe(
+      const runsReload$ = this.onDashboardLoad$.pipe(
+        map(([, routeKind]) => routeKind),
+        switchMap((routeKind) => {
+          if (routeKind !== RouteKind.COMPARE_EXPERIMENT) {
+            return of([]);
+          }
+
+          return this.store.select(getExperimentIdToAliasMap).pipe(
+            distinctUntilChanged((beforeAliasDict, afterAliasDict) => {
+              const entries = Object.entries(beforeAliasDict);
+              const afterAliasMap = new Map(Object.entries(afterAliasDict));
+              if (entries.length !== afterAliasMap.size) {
+                return false;
+              }
+              for (const [experimentId, alias] of entries) {
+                if (afterAliasMap.get(experimentId) !== alias) {
+                  return false;
+                }
+              }
+              return true;
+            }),
+            // HACK: arbitrary microtask delay.
+            // aliasChange leads to route changes and route changes causes
```

Comment:
aliasChange does not exist as a symbol, but maybe you just want "An alias change..."